test/karma.config.coffee

```diff
@@ -1,12 +1,72 @@
+webpack = require('webpack')
+
 module.exports = (config) ->
   config.set
     basePath: '../'
-    frameworks: ['mocha']
+    frameworks: ['mocha', 'chai', 'chai-sinon', 'phantomjs-shim']
     browsers: ['PhantomJS']
-    # browsers: ['Chrome']
-    reporters: ['mocha']
-    # plugins: ['karma-mocha', 'karma-phantomjs-launcher', 'karma-mocha-reporter']
+    reporters: ['mocha', 'coverage']
+
     files: [
-      {pattern: 'test/phantomjs-shims.js'}
-      {pattern: '.tmp/all-tests.js'}
+      'test/all-tests.coffee'
+      'test/all-source-files.coffee'
+    ]
```

Comment:
This controls what appears in the coverage report.  It's possible to include src files directly, for instance. `src/components/*.cjsx` would work.  However that fails when web pack also includes the file as part of a spec.  Webpack complains about:

ERROR in ./src/components/course-settings/add-student.cjsx
Module not found: Error: a dependency to an entry point is not allowed
 @ ./src/components/course-settings/add-student.cjsx 11:13-38